docs/releasing/publishing.md

```diff
@@ -1,55 +1,62 @@
   - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
   - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
   - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
   - update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
 
-2. Add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
-
-3. Write a summary to post on Slack after the release. For example:
-
-    🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @SLACK-NAME for helping with this release. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
+3. Add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
 
-    Remember that Prototype Kit needs its own summary, since it has its own release notes.  
+4. Content designer to:
+  - write announcements to post on Slack after we release:
+    - GOV.UK Frontend (for example, [draft comms for the cookie banner component](https://docs.google.com/document/d/1jVyMB7i94NOeflWaf3kE4Q4APMXGfluK3rOh74IHO08/edit))
+    - GOV.UK Prototype kit
+  - check who the release’s contributors are and if we have consent to include their name
 
-4. Ask a technical writer or a content designer to review the summary.
+5. Technical writer to finalise draft of release notes and get 2i on them.
 
-5. Before you proceed to the next section, confirm with the appropriate person that the release notes and any related pull requests are ready to publish. Depending on the issue, the appropriate person may be the technical writer, the content designer, or a developer. If you’re unsure who it is, then ask within the [#govuk-design-system](https://gds.slack.com/archives/CAF8JA25U) Slack channel.
+6. Content designer/Designer to update community backlog with rationale for any decisions we made.
 
-# Steps you must complete to release a new version of GOV.UK Frontend
+7. Coordinate the sign-off when you are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the []#design-system-team-channel](https://app.slack.com/client/T8GT9416G/G011V7RAAKY) to confirm sign-off from:
+  - Content and designers for guidance, examples and community backlog decision rationale
+  - Technical writer and developers for Nunjucks macros
+  - Developers for changes to GOV.UK Frontend
+  - Technical writer for release notes
+  - Content/Community Manager for announcements and engagement activities
```

Comment:
I think this should say 'Content designer/Community manager'

Reply:
As with the previous comment, 'Content' here was meant as shorthand for 'content designer + tech writer.' Would it be ok to write "Content designer/Community manager/Technical writer"? I don't always draft comms, but it's something I do occasionally (like with the ongoing branch-renaming work).

Reply:
```suggestion
  - Content designer/Community manager/Technical writer for announcements and engagement activities
```

Reply:
Yes, sounds good 👍